docs/source/doctest_fixtures.py

```diff
@@ -92,6 +93,10 @@
 # Disable wandb
 os.environ['WANDB_MODE'] = 'disabled'
 
+# Disable MosaicMLLogger
+os.environ['MOSAICML_PLATFORM'] = 'false'
+os.environ.pop('MOSAICML_ACCESS_TOKEN_FILE', None)
```

Comment:
This has side-effects....

but maybe its fine

Reply:
we can remove I don't think we need this anymore.